data/CentOS/6.yaml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
can you please add the missing `\n` at the end of this line? (same applies for the other files)

Reply:
in the past we also had issues with empty yaml files, but I cannot remember what broke and which conditions :(